src/main/java/io/pakland/mdas/githubstats/domain/ports/UseCase.java

```diff
@@ -0,0 +1,7 @@
+package io.pakland.mdas.githubstats.domain.ports;
+
+public interface UseCase {
+
+    void execute();
+
+}
```

Comment:
This is something that we have discussed a bit with the teacher. The thing is that having an interface to define a contract for the `UseCase` is redundant, basically because it is really hard of using some sort of dependency injection so that this interface comes in to play.  
I would remove it.

Reply:
Removed! ✅

Reply:
understood, this interface I though was deleted sorry. I was using it at the first, but then removed. Maybe I made a mistake no commiting the deleted file.